doc/persistence.rst

```diff
@@ -19,6 +19,10 @@ Usecase
 pickle to work efficiently on Python objects containing large data, in
 particular large numpy arrays.
 
+..
+  >>> # For running doctests, users may ignore this block
```

Comment:
Same comment, move that to the top of the file and use a comment like "py.test fixture to run doctests"